src/common/components/header/index.js

```diff
@@ -11,34 +12,55 @@ export default class Header extends Component {
     const { authenticated, user } = this.props;
 
     return (
-      <div className={ styles.header }>
-        <nav className={ styles.container }>
-          <Link className={ styles.logo } to="/">
-            <img src={ wordmark } alt="Snapcraft.io" height={ 28 } />
-          </Link>
-          { authenticated
-            ?
-              <div className={ styles.sideNav }>
-                { user &&
-                  <span className={ styles.username } >
-                    Hi, {user.name || user.login}
-                  </span>
-                }
-                <a
-                  href="/auth/logout"
-                  className={ styles.link }
-                  onClick={ this.onLogoutClick.bind(this) }
-                >
-                  Sign out
-                </a>
-              </div>
-            :
-              <div className={ styles.sideNav }>
-                <a href="/auth/authenticate" className={ styles.link }>Sign in</a>
-              </div>
-          }
-        </nav>
-      </div>
+      <header id="navigation" className={ style['p-navigation--light'] }>
+        <div className={ containerStyles.wrapper }>
+          <div className={ style['p-navigation__banner'] }>
+            <div className={ style['p-navigation__logo'] }>
+              <a className={ style['p-navigation__link'] } href="https://snapcraft.io">
+                <img className={ style['p-navigation__image'] } src={ wordmark } alt="Snapcraft.io" width={ 148 } />
+              </a>
+            </div>
+            <a href="#navigation" className={ style['p-navigation__toggle--open'] } title="menu">Menu</a>
+            <a href="#navigation-closed" className={ style['p-navigation__toggle--close'] } title="close menu">Close menu</a>
+          </div>
+          <nav className={ style['p-navigation__nav'] } role="menubar">
+            <ul className={ style['p-navigation__links']} role="menu">
+              <li className={ style['p-navigation__link'] } role="menuitem">
+                <a href="https://snapcraft.io/store/">Store</a>
```

Comment:
You need to add `className={ style['p-link--external'] }`

Reply:
oh I just read your comment
> should Store be external link on BSI? If so it looks strange to have external link first in the navigation. Also, if both sites have same navigation maybe we should treat them as same 'site', so both Store and Build should not be external links?

Reply:
Yes, I know... I'm asking about it in comment above, because it looks quite weird to have Store as an external link.

<img width="1080" alt="screen shot 2018-01-19 at 17 02 26" src="https://user-images.githubusercontent.com/83575/35159472-9e2407ec-fd3a-11e7-8f0e-cbd733d4a7fb.png">

So maybe this would need some other changes, like changing the order of links or treating Store and Build and not external to each other... UX input is needed here.